test-suite/exonum-py-tests/exonum_tests/api.py

```diff
@@ -0,0 +1,29 @@
+"""Tests for Exonum API."""
+
+import unittest
+import time
+
+from exonum_client import ExonumClient
+
+from suite import run_4_nodes, assert_processes_exited_successfully
+
+
+class ApiTest(unittest.TestCase):
+    """Tests for Exonum API."""
+
+    def test_health_info(self):
+        """Tests the `health_info` endpoint."""
```

Comment:
Did you mean `healthcheck` ?
https://github.com/exonum/exonum/blob/master/exonum/src/api/node/public/system.rs#L153
```suggestion
        """Tests the `healthcheck` endpoint."""
```